hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -807,6 +821,27 @@ class TestHoodieSparkSqlWriter {
     }).count())
   }
 
+  /**
+   * Test case for deletion of partitions using wildcards
+   * @param partition the name of the partition(s) to delete
+   */
+  @ParameterizedTest
+  @ValueSource(strings = Array("2015/03/*", "*5/03/1*"))
```

Comment:
can we make the arg list as 1. input and expected partition list. 
for eg, we can test 
"2015/03/*","2016/03/15"
"*5/03/1*","2016/03/15"
"2016/03/*","2015/03/16,2015/03/17"